### PR TITLE
[All] Update to Compose 1.1.1

### PR DESCRIPTION
**Pull request by IanGClifton**
This also updates spotless to 6.3, ktlint to 0.44.0, and some other
minor/bugfix library updates.

### CODE REVIEW

File: Jetsurvey/buildSrc/src/main/java/com/example/compose/jetsurvey/buildsrc/dependencies.kt [hunk 1]
```diff
@@ -17,17 +17,15 @@
...
-    const val androidGradlePlugin = "com.android.tools.build:gradle:7.1.0"
+    const val androidGradlePlugin = "com.android.tools.build:gradle:7.1.2"
     const val jdkDesugar = "com.android.tools:desugar_jdk_libs:1.1.5"
 
     const val junit = "junit:junit:4.13"
 
-    const val material = "com.google.android.material:material:1.3.0"
-
     object Accompanist {
         const val version = "0.23.0"
```
Comment (bentrengrove) [inline Jetsurvey/buildSrc/src/main/java/com/example/compose/jetsurvey/buildsrc/dependencies.kt:30]: Accompanist will need an update to 0.23.1

File: Jetchat/buildSrc/src/main/java/com/example/compose/jetchat/buildsrc/dependencies.kt [hunk 2]
```diff
@@ -57,7 +57,7 @@ object Libs {
 
         object Compose {
```
Comment (ricknout) [inline Jetchat/buildSrc/src/main/java/com/example/compose/jetchat/buildsrc/dependencies.kt:58]: Please update Material 3 to 1.0.0-alpha06

File: Jetsurvey/buildSrc/src/main/java/com/example/compose/jetsurvey/buildsrc/dependencies.kt (hunk 1 above)
Comment (IanGClifton) [reply Jetsurvey/buildSrc/src/main/java/com/example/compose/jetsurvey/buildsrc/dependencies.kt]: Done, thanks!

File: Jetchat/buildSrc/src/main/java/com/example/compose/jetchat/buildsrc/dependencies.kt (hunk 2 above)
Comment (ricknout) [reply Jetchat/buildSrc/src/main/java/com/example/compose/jetchat/buildsrc/dependencies.kt:58]: Nope, I meant https://maven.google.com/web/index.html#androidx.compose.material3:material3:1.0.0-alpha06. For some reason I couldn't comment inline on line 78

Comment (IanGClifton) [reply Jetchat/buildSrc/src/main/java/com/example/compose/jetchat/buildsrc/dependencies.kt:58]: My mistake, thanks for clarifying! I updated it to alpha06. The one change affecting jetchat that I saw was NavigationDrawer was renamed to ModalNavigationDrawer, so I went ahead and updated that too.

**Review by bentrengrove — APPROVED** [review]
LGTM! Assuming we wanted to keep the non-compose Material upgrade to an alpha version @ricknout?

**Review by ricknout — APPROVED** [review]

**Review by Gurupreet — APPROVED** [review]

**Pull request merged**